### PR TITLE
RPET-964 returning to awaiting service consideration from service application not approved

### PR DESCRIPTION
**Pull request by anatolysan**
https://tools.hmcts.net/jira/browse/RPET-964

https://github.com/hmcts/div-case-orchestration-service/pull/1428

### CODE REVIEW

File: definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField.json [hunk 1]
```diff
@@ -19867,6 +19867,48 @@
     "UserRole": "citizen",
     "CRUD": "R"
   },
+  {
+    "LiveFrom": "23/07/2020",
```
Comment (yarpo) [inline definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField.json:19871]: `LiveFrom` is ignored by CCD, but it helps a little bit to investigate what happened when (even though we can get more reliable info from git blame).

Can you please update all `LiveFrom` with current date?

File: definitions/divorce/json/CaseEvent/CaseEvent.json [hunk 2]
```diff
@@ -3263,8 +3263,8 @@
     "Description": "Service application received",
     "DisplayOrder": 11,
     "PreConditionState(s)": "AosStarted;AosOverdue;ServiceApplicationNotApproved;AosAwaiting;AosDrafted;Submitted;Issued;AwaitingDocuments",
-    "PostConditionState": "AwaitingServicePayment",
-    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/received-service-added-date",
+    "PostConditionState": "*",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/service-application-received",
     "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
```
Comment (yarpo) [inline definitions/divorce/json/CaseEvent/CaseEvent.json:3268]: you can remove `RetriesTimeoutURLAboutToSubmitEvent`. It's redundant. CCD ignores it.

File: definitions/divorce/json/CaseEventToFields/CaseEventToFields.json [hunk 3]
```diff
@@ -9665,6 +9665,19 @@
     "PageColumnNumber": 1,
     "ShowSummaryChangeOption": "No"
   },
+  {
+    "LiveFrom": "23/07/2020",
```
Comment (yarpo) [inline definitions/divorce/json/CaseEventToFields/CaseEventToFields.json:9669]: please update date.

File: package.json [hunk 4]
```diff
@@ -19,7 +19,7 @@
       "ccdUrl": "http://ccd-data-store-api-demo.service.core-compute-demo.internal"
     },
     "aat": {
-      "cosUrl": "http://div-cos-aat.service.core-compute-aat.internal",
+      "cosUrl": "http://div-cos-pr-1428.service.core-compute-preview.internal",
```
Comment (yarpo) [inline package.json:22]: please make sure you'll revert this update. It must not get to master.

File: definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField.json (hunk 1 above)
Comment (anatolysan) [reply definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField.json]: :white_check_mark:

File: definitions/divorce/json/CaseEvent/CaseEvent.json (hunk 2 above)
Comment (anatolysan) [reply definitions/divorce/json/CaseEvent/CaseEvent.json]: :white_check_mark:

File: definitions/divorce/json/CaseEventToFields/CaseEventToFields.json (hunk 3 above)
Comment (anatolysan) [reply definitions/divorce/json/CaseEventToFields/CaseEventToFields.json]: :white_check_mark:

**Review by alanwilliamsmoj — APPROVED** [review]

**Review by ruban72 — APPROVED** [review]

**Pull request closed without merging**